src/components/QuickOpenModal.js

```diff
@@ -136,31 +136,31 @@ export class QuickOpenModal extends Component<Props, State> {
     if (item == null) {
       return;
     }
-    const { selectSource, selectedSource, query, searchType } = this.props;
+    const { selectLocation, selectedSource, query, searchType } = this.props;
     if (this.isSymbolSearch()) {
       if (selectedSource == null) {
         return;
       }
-      selectSource(selectedSource.get("id"), {
-        location: {
-          ...(item.location != null ? { line: item.location.start.line } : {})
-        }
-      });
+      const resultItemLocation = { sourceId: selectedSource.get("id") };
+      if (item.location != null) {
```

Comment:
This might be rewritten in a nicer way similar to: 
` ...(!isNaN(columnNumber) ? { column: columnNumber } : null)`